ruby-challenges.rb

```diff
@@ -3,32 +3,81 @@
 
 # --------------------1) Create a method that takes in a number and determines if the number is even or odd. Use the test variables provided.
 
-reposts1 = 7
+# reposts1 = 7
 # Expected output: '7 is odd'
-reposts2 = 42
+# reposts2 = 42
 # Expected output: '42 is even'
-reposts3 = 221
+# reposts3 = 221
 # Expected output: '221 is odd'
 
+# def odd_number(reposts3)
+#     if reposts3 % 2 != 0
+#      "#{reposts3} is odd"
+# else 
+#     reposts3 %2 == 0
```

Comment:
This line is not necessary. Also, name your params a different name than given variables as best practice.